wolfcrypt/src/rsa.c

```diff
@@ -1736,6 +1736,60 @@ int wc_RsaUnPad_ex(byte* pkcsBlock, word32 pkcsBlockLen, byte** out,
     return ret;
 }
 
+int hash2mgf(enum wc_HashType hType)
+{
+    switch (hType) {
+    case WC_HASH_TYPE_SHA:
+#ifndef NO_SHA
+        return WC_MGF1SHA1;
+#else
+        WOLFSSL_MSG("Unrecognized or unsupported hash function");
```

Comment:
Can you refactor this to eliminate the duplicate MSG and return code logic? Use break and single unrecognized exit. Also maybe rename this function to include wc_Rsa since its a "local" function?